### PR TITLE
Move percentage to mod.rs

### LINKED ISSUE #18001: Move `Percentage` value structs to mod.rs
Currently `Percentage` structs are in `components/style/values/{specified,computed}/length.rs`. They have been put to this file probably because many length values are using it. But they are not related to length by themselves. We should move them to `components/style/values/{specified,computed}/mod.rs` instead and make sure all `./mach build -d`, `./mach build-geckolib` and `./mach test-unit -p style` compiles.

### PR DESCRIPTION
**Pull request by DarkDrek**
The tests in the issue worked for me locally.

---

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [X] These changes fix #18001 (github issue number if applicable).

- [X] These changes do not require tests because they are just refactoring

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/18003)

### CODE REVIEW

File: components/style/values/computed/length.rs [hunk 1]
```diff
@@ -19,33 +19,6 @@ use values::specified::length::ViewportPercentageLength;
...
-    }
-}
-
-impl ToCss for Percentage {
-    fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
-        write!(dest, "{}%", self.0 * 100.)
-    }
-}
-
 impl ToComputedValue for specified::Percentage {
```
Comment (canaltinova) [inline components/style/values/computed/length.rs]: We can move this impl as well.

**Review by canova — COMMENTED** [review]
Looks good to me. Just one more thing.
Thanks for working on this!

**Comment by KiChjang** [comment]
@DarkDrek Could you squash your commits into one?

**Comment by canaltinova** [comment]
Thanks again!
@bors-servo r+

**Pull request merged**